src/components/four-up.tsx

```diff
@@ -428,6 +428,8 @@ export class FourUpComponent extends BaseComponent<IProps, IState> {
       const { group } = this.props;
       this.tabUIModel?.getDocumentGroup(group.id)?.closePrimaryDocument();
 
+      Logger.log(LogEventName.TOOLBAR_FOUR_UP_TOOL);
```

Comment:
Not sure, but should this also be a call to `logDocumentOrSectionEvent` ?

Reply:
There is a document or section focused at the time the four up is clicked but the event is about returning to 4-up view so it really isn't about the document/section so I just used a raw event.